### PR TITLE
Add initial events data and update script

### PR DESCRIPTION
**Pull request by lauravuo**
## Purpose

* Refactored folder structure so that db scripts are in tools folder
* Added `data`-folder that contains the service events as json. The events are rendered to app main view (as a first test).
* Added script (`fetch`) for updating the events json from Firebase. The script is run daily via GitHub actions (can be triggered also from the actions UI). Logic for the data updates will be tuned more later.
* The `fetch`-script also generates an `index.js`-file for easily include the events to React-components. Currently it includes all available events, but in production-phase the purpose is to include only the events in current month and onwards. 

## Related Issues

I will create later issues for the missing functionality such as unit tests for the db scripts and issues mentioned above.

## How to test

DB events inclusion to React: Start app in localhost and see that events are listed with index and title to main view.
DB scripts: Open Actions tab and see that the nightly run for workflow `fetch-data` has succeeded.

### CODE REVIEW

File: .github/workflows/fetch.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - name: remove creds
+        run: rm ./tools/firebase/creds.json
+      # TODO: run tests and push only if they succeed
+      - name: commit changes
+        run: |
+          git config --global user.name fetch-bot
+          git config --global user.email fetch-bot
+          git add .
+          git commit -a -m "Add updates from db."
+          git push origin main
```
Comment (eevajonnapanula) [inline .github/workflows/fetch.yml:36]: Nit: There should be a newline

**Review by eevajonnapanula — COMMENTED** [review]
Looking good! I left two nit pick comments, feel free to ignore them if you feel like it 😄 

Everything seems to work fine from the React-side, but I can't see a passing workflow run in the Actions. However, I'm not sure if it's becase this is not in main? Should the nightly run happen when it's not in the main branch?

**Comment by lauravuo** [comment]
You're @eevajonnapanula right! The scheduled action will be run only with the default branch. I add the push event temporarily now that we can check if the action passes.

**Comment by lauravuo** [comment]
Ok, so when testing in the correct branch, [the job succeeds](https://github.com/kaupunginnaiset/alman-akka/actions/runs/1997243091).

Relevant parts are `update data` and `commit changes` steps: first script fetches the changed entries from db and commit step pushes the new data files to repository.

image

**Pull request merged**